options/locale/locale_en-US.ini

```diff
@@ -1029,8 +1029,8 @@ issues.dependency.add = Add dependency…
 issues.dependency.cancel = Cancel
 issues.dependency.remove = Remove
 issues.dependency.remove_info = Remove this dependency
-issues.dependency.added_dependency = `<a href="%[1]s">%[2]s</a> added a new dependency %[3]s`
-issues.dependency.removed_dependency = `<a href="%[1]s">%[2]s</a> removed a dependency %[3]s`
+issues.dependency.added_dependency = `added a new dependency %s`
```

Comment:
In order to maintain certain degree of compatibility, we should still generate parameters `%[1]s`, `%[2]s` and `%[3]s` in the corresponding function. So this string should be:
```suggestion
issues.dependency.added_dependency = `added a new dependency %[3]s`
```

Reply:
there is no really point in that as all translations will be lost anyway

Reply:
@lafriks How come? If Spanish/French/Polish are not updated, the old string would still work. Otherwise the UI will break in those languages.

Reply:
I believe that once crowdin receives an update for a string it'll blank the string from other languages (although I'm not 100% sure of that)

Reply:
not sure about that, the identifier will not change. I would rather log to croudwin and correct all translations after merge

somebody will tidy those blank params in the future and all translators will have to translate it once again.

Reply:
@guillep2k as @techknowlogick already described as soon as this PR is merged crowdin will reset and commit this key for all other languages

Reply:
Oh, I didn't know that.